processing/src/main/java/io/druid/query/aggregation/first/DoubleFirstAggregatorFactory.java

```diff
@@ -49,15 +47,11 @@
 
 public class DoubleFirstAggregatorFactory extends NullableAggregatorFactory<ColumnValueSelector>
 {
-  public static final Comparator VALUE_COMPARATOR = (o1, o2) -> Doubles.compare(
-      ((SerializablePair<Long, Double>) o1).rhs,
-      ((SerializablePair<Long, Double>) o2).rhs
-  );
+  public static final Comparator VALUE_COMPARATOR =
+      Comparator.comparingDouble(o -> ((SerializablePair<Long, Long>) o).rhs);
```

Comment:
should this be SerializablePair<Long, Double> instead of SerializablePair<Long,Long> ?